src/TodosContext/TodosContext.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+import { TodosContextType } from '../types/TodosContextType';
+import { Todo } from '../types/Todo';
+import { SortType } from '../types/SortType';
+import * as postService from '../api/todos';
+import { USER_ID } from '../utils/USER_ID';
+
+export const TodosContext = React.createContext<TodosContextType>({
+  todos: [],
+  setTodos: () => { },
+  errorMessage: '',
+  setErrorMessage: () => { },
+  preparedTodos: [],
+  sortQuery: '',
+  setSortQuery: () => {},
+  tempTodo: {
+    id: 0,
+    userId: 0,
+    title: '',
+    completed: false,
+  },
+  setTempTodo: () => {},
+  todosInProcess: [],
+  setTodosInProcess: () => {},
+  completedTodos: [],
+});
```

Comment:
pls try to reduce the amount of props passed to a component, it's bad practive when u have to many props